src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
I'm struggling with how can I place a judgment prior to testing similar patches to better understand whether any domain-specific knowledge I have applies.  I understand there is some flexibility in our patching process, yet at the same time I'm attempting (and failing) to provide a satisfactory answer to, "Is it accepted, rejected, or deferred until concerns about functionality/performance are addressed?" as my intent was:
* If anti_lmr7 is successful, accept both patches
* If it fails, do additional experiments to understand the cause of failure

as the context surrounding the code change (including the immediately following `else` block) has me concerned about the possibility of this and/or other changes overfitting (to a local maximum Elo), a situation which is difficult to remedy.

Normally I wouldn't hesitate to merge a PR, but here I can't guarantee that I can improve an entire +5 Elo over `anti_lmr5`; maybe I can only improve +9 (or fewer) Elo over `master` with a similar patch, because I lacked the foresight to provide feedback on code I overlooked prior to LTC.